crates/compiler/builtins/bitcode/src/list.zig

```diff
@@ -462,10 +481,12 @@ fn listAppend(list: RocList, alignment: u32, element: Opaque, element_width: usi
 
 pub fn listPrepend(list: RocList, alignment: u32, element: Opaque, element_width: usize) callconv(.C) RocList {
     const old_length = list.len();
-    var output = list.reallocate(alignment, old_length + 1, element_width);
+    // TODO: properly wire in update mode.
```

Comment:
What's update mode?

Reply:
Not actually thinking about seamless slices in this case. It actually would be an optimization for regular list.

If a list is know unique at compile time, it allows for optimizations where we can just directly take the unique path of a function call instead of checking the refcount. At least that is my understanding